pkg/minikube/kubeconfig/config.go

```diff
@@ -59,7 +60,7 @@ func GetConfigPath() string {
 }
 
 // Cache system admin entries to be used to run oc commands
-func CacheSystemAdminEntries(systemEntriesConfigPath, clusterName string) error {
+func CacheSystemAdminEntries(systemEntriesConfigPath, clusterName string, isVersion3_6 bool) error {
```

Comment:
I am little confused. The change in logic for all versions of Origin > = 3.6 , right? But the code seems to be on only 3.6 version..